src/components/pages/novu-framework/code/code.jsx

```diff
@@ -113,23 +105,23 @@ const Code = () => {
             {code}
           </SyntaxHighlighter>
           <StaticImage
-            className="!absolute pointer-events-none bottom-[-85px] right-0 z-0 w-[1252px] lg:bottom-[-60px] lg:w-[1044px] md:!hidden"
+            className="!absolute pointer-events-none bottom-[-73px] right-0 z-0 w-[1252px] lg:bottom-[-46px] lg:w-[1044px] md:!hidden"
```

Comment:
Some padding fixes to account for the additional line. Tested across all responsive widths